pallets/pallet-bonded-coins/src/curves/polynomial.rs

```diff
@@ -1,3 +1,24 @@
+/// Polynomial bonding curve implementation.
+///
+/// This module provides an implementation of a polynomial bonding curve. The current implementation supports a square bonding curve, where the integral is precomputed.
+/// The cost function for the polynomial bonding curve is defined as:
+/// C(s) = m * s^3 + n * s^2 + o * s,
+/// where:
+/// - `s` is the supply of assets,
+/// - `m` is the coefficient for the cubic part,
+/// - `n` is the coefficient for the quadratic part,
+/// - `o` is the coefficient for the linear part.
```

Comment:
it probably helps to explain what m, n, and o stand for in the price function, and how to derive the parameters of the price function from the given values

Reply:
I added more explanation in: [8f8664a](https://github.com/KILTprotocol/kilt-node/pull/801/commits/8f8664a534ff38f5e7f491145f18f69f0eb1019f) 

Do you think it is clearer now?